src/libraries/Microsoft.Extensions.Logging/gen/DiagDescriptors.cs

```diff
@@ -0,0 +1,171 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Extensions.Logging.Generators
+{
+    internal static class DiagDescriptors
```

Comment:
(nit) I'm not sure if this is a common naming pattern, but typically we don't use abbreviations in class names.

=> `DiagnosticDescriptors`